pkg/openshift-sdn/informers.go

```diff
@@ -32,7 +32,12 @@ func (sdn *OpenShiftSDN) buildInformers() error {
 	if err != nil {
 		return err
 	}
-	kubeClient, err := kubernetes.NewForConfig(kubeConfig)
+
+	protoKubeConfig := rest.CopyConfig(kubeConfig)
+	protoKubeConfig.AcceptContentTypes = "application/vnd.kubernetes.protobuf,application/json"
+	protoKubeConfig.ContentType = "application/vnd.kubernetes.protobuf"
+
+	kubeClient, err := kubernetes.NewForConfig(protoKubeConfig)
```

Comment:
@squeed hint: we usually use `protoKubeClient` as using this client for any CR's will result in catastrophic failure :-)